### PR TITLE
db/component-width-unify

### PR DESCRIPTION
**Pull request by sebastianbochan**

### CODE REVIEW

File: samples/dashboards/demos/layout-nested/demo.js [hunk 1]
```diff
@@ -42,7 +42,7 @@ Dashboards.board('container', {
                                         cells: [{
                                             id: 'dashboard-col-layout-2a'
                                         }, {
-                                            id: 'dashboard-col-layout-2b'
+                                            id: 'dashboard-col-layout-2b',
```
Comment (pawelfus) [inline samples/dashboards/demos/layout-nested/demo.js]: unnecessary comma 😬

**Review by pawelfus — APPROVED** [review]
Thanks!

File: ts/Dashboards/Layout/Cell.ts [hunk 2]
```diff
@@ -502,14 +509,31 @@ class Cell extends GUIElement {
...
+        if (typeof width === 'number') {
+            convertedWidth = width + 'px';
+        } else if (/px/.test(width)) {
+            convertedWidth = width;
+        } else {
+            convertedWidth = GUIElement.getPercentageWidth(width) || '';
+        }
+
+        return convertedWidth;
+    }
```
Comment (karolkolodziej) [inline ts/Dashboards/Layout/Cell.ts:528]:
```suggestion
    private convertWidthToValue(width: number|string): string {
        if (typeof width === 'number') {
            return width + 'px';
        }
        if (/px/.test(width)) {
            return width;
        }

        return GUIElement.getPercentageWidth(width) || '';
    }
```

**Review by karolkolodziej — CHANGES_REQUESTED** [review]

**Review by pawellysy — APPROVED** [review]

File: ts/Dashboards/Layout/Cell.ts (hunk 2 above)
Comment (sebastianbochan) [reply ts/Dashboards/Layout/Cell.ts]: Nice!

**Review by karolkolodziej — APPROVED** [review]

**Pull request merged**